src/cloud/components/onboarding/useGetUserStatus.ts

```diff
@@ -83,29 +83,67 @@ export const getUserStatus = (table: Table): USER_PILOT_USER_STATUS[] => {
   return dataStates
 }
 
+export const queryUsage = async (
+  orgID: string,
+  queryLength?: string
+): Promise<Table> => {
+  let csvToParse = ''
+  if (getOrgsUsage) {
+    if (isFlagEnabled('newUsageAPI')) {
+      const usage = await getOrgsUsage({
+        orgID,
+        query: {
+          start: queryLength ?? '-30d',
+        },
+      })
+      if (usage?.status === 200) {
+        console.warn(`Usage: ${usage}`)
+        csvToParse = usage.data?.trim().replace(/\r\n/g, '\n')
+        console.warn(`CSVTOPARSE: ${csvToParse}`)
+      }
+    } else {
+      csvToParse = usageStatsCsv
+    }
+    const {table} = fromFlux(csvToParse)
+
+    return table
+  }
+}
+
+export const getUserWriteLimitHits = async (
+  orgID: string,
+  queryLength?: string
+): Promise<number> => {
+  try {
+    const table = await queryUsage(orgID, queryLength ?? '-1h')
```

Comment:
if this the only place `queryUsage` is being called? If so it seems like the default queryLength can just take place in the query if the `queryLength` is undefined right? You've got `-1h` and `-30d`, just a thought

Reply:
at _this_ point yes, but i wanted this to be a bit more extendable so ive passed a param with a default